src/biomappings/resources/incorrect.tsv

```diff
@@ -599,6 +599,7 @@ mesh	C477269	3 beta-hydroxy-delta 5-steroid dehydrogenase, rat	skos:exactMatch	g
 mesh	C509674	marinocine protein, Marinomonas mediterranea	skos:exactMatch	go	GO:0033736	L-lysine 6-oxidase activity	semapv:ManualMappingCuration	orcid:0000-0001-9439-5346			
 mesh	C531673	Familial ependymoma	skos:exactMatch	doid	DOID:5509	pediatric ependymoma	semapv:ManualMappingCuration	orcid:0000-0003-1307-2508			
 mesh	C531763	Irritable heart	skos:exactMatch	doid	DOID:0111154	postural orthostatic tachycardia syndrome	semapv:ManualMappingCuration	orcid:0000-0003-4423-4370			
+<<<<<<< HEAD
```

Comment:
@svempati21 you have to be extra careful in this repository when merging. This needs to be fixed